services/web/server/src/simcore_service_webserver/groups_handlers.py

```diff
@@ -166,7 +166,7 @@ async def add_group_user(request: web.Request):
 
     new_user_id = new_user_in_group["uid"] if "uid" in new_user_in_group else None
     new_user_email = (
-        new_user_in_group["email"] if "email" in new_user_in_group else None
+        new_user_in_group["email"].lower() if "email" in new_user_in_group else None
```

Comment:
TIP:
Another nice thing to do here is the following.

```python
from pydantic import parse_obj_as

parse_obj_as(LowerCaseEmailStr, new_user_in_group["email"])
```

This way you apply all validators to the field.